dm/tests/downstream_more_column/data/db1.prepare.sql

```diff
@@ -2,6 +2,10 @@ drop database if exists `downstream_more_column`;
 drop database if exists `downstream_more_column1`;
 create database `downstream_more_column1`;
 use `downstream_more_column1`;
-create table t1 (c1 int, c2 int, c3 int, primary key(c1));
+create table t1 (c1 int, c2 int, c3 int comment '1;
```

Comment:
Can we test with dumpling format, this stmt will be format by ShowCreateTable as
```
CREATE TABLE `t1` (
  `c1` int(11) NOT NULL,
  `c2` int(11) DEFAULT NULL,
  `c3` int(11) DEFAULT NULL COMMENT '1;\n2',
  PRIMARY KEY (`c1`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1
```